napari/_qt/qt_viewer.py

```diff
@@ -1028,6 +1028,16 @@ def _set_drag_status(self):
             'Hold <Alt> key to open plugin selection. Hold <Shift> to open files as stack.'
         )
 
+    def _image_from_clipboard(self):
+        """Insert image from clipboard as a new layer if clipboard contains image."""
+        cb = QGuiApplication.clipboard()
+        if cb.mimeData().hasImage():
```

Comment:
Maybe worth an `else` with a notification, sort of like @kephale has here
https://github.com/kephale/napari-clipboard/blob/b7b9c71df6eb4703330573defa5b5909181fa6ba/src/napari_clipboard/_widget.py#L34

Something like "Clipboard doesn't contain an image; unable to make a new Image layer."

Reply:
done